### ISSUE TITLE
2.0.3 tests fail on fedora/pbuilder

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1108]
I first saw this in my local pbuilder after putting together the 2.0.3 release. Things worked fine on debian's buildd (surprisingly), but they're breaking on the Fedora34 builder:

```
5: get_tty_fd:848:Error opening /dev/tty (No such device or address)
5: Defaulting to 24x80 (output is not to a terminal)
5: ncplane_new_internal:367:Created new 24x80 plane "std" @ 0x0
5: ncplane_new_internal:367:Created new 1x1 plane "" @ 1x1
5: ild/BUILD/notcurses-2.0.3/tests/visual.cpp:626: ERROR: CHECK( 1 == dimy ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:627: ERROR: CHECK( 1 == dimx ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:631: ERROR: CHECK( 0 == strcmp(egcs[i], egc) ) is NOT correct!
5:   values: CHECK( 0 == 1 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:633: ERROR: CHECK( 0x111111 == channels_fg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 1118481 == 0 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:634: ERROR: CHECK( 0xdddddd == channels_bg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 14540253 == 0 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:626: ERROR: CHECK( 1 == dimy ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:627: ERROR: CHECK( 1 == dimx ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:631: ERROR: CHECK( 0 == strcmp(egcs[i], egc) ) is NOT correct!
5:   values: CHECK( 0 == 1 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:633: ERROR: CHECK( 0x111111 == channels_fg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 1118481 == 0 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:634: ERROR: CHECK( 0xdddddd == channels_bg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 14540253 == 0 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:626: ERROR: CHECK( 1 == dimy ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:627: ERROR: CHECK( 1 == dimx ) is NOT correct!
5:   values: CHECK( 1 == 2 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:631: ERROR: CHECK( 0 == strcmp(egcs[i], egc) ) is NOT correct!
5:   values: CHECK( 0 == 1 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:633: ERROR: CHECK( 0x111111 == channels_fg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 1118481 == 0 )
5: 
5: /builddir/build/BUILD/notcurses-2.0.3/tests/visual.cpp:634: ERROR: CHECK( 0xdddddd == channels_bg_rgb(channels) ) is NOT correct!
5:   values: CHECK( 14540253 == 0 )
5: 
5: [?1h=[?1h=[?1h=[?1h=[?1h=[?1h=[?1h=[?1h================================================================================
5: [doctest] test cases:    39 |    38 passed |   1 failed | 1 skipped
5: [doctest] assertions: 12334 | 12224 passed | 110 failed |
5: [doctest] Status: FAILURE!
5/5 Test #5: notcurses-tester .................***Failed    1.00 sec
```

same place on both.

**Comment by dankamongmen** [comment]
I'm pretty sure this was just failure to enforce UTF8 in these tests. Quadblitter thus fell back to ASCII, which explains exactly the results we've seen here.

**Comment by dankamongmen** [comment]
Fixed for 2.0.4.
